fuzz/fuzz_targets/fuzz_process_rand.rs

```diff
@@ -0,0 +1,257 @@
+use honggfuzz::fuzz;
```

Comment:
Same comment as previous fuzz target. Also looking for confirmation that the seeded random is guaranteed to be fixed.